controllers/operands/kubevirt.go

```diff
@@ -294,7 +324,75 @@ func hcWorkloadUpdateStrategyToKv(hcObject *hcov1beta1.HyperConvergedWorkloadUpd
 	return kvObject
 }
 
-func getKVConfig(hc *hcov1beta1.HyperConverged) (*kubevirtcorev1.KubeVirtConfiguration, error) {
+func getRatesFromConfigMap(configMap *corev1.ConfigMap) (*kubevirtcorev1.ReloadableComponentConfiguration, *RateTuningError) {
+
+	qpsRead, ok := configMap.Data[qps]
+
+	if !ok {
+
+		return nil, NewTuningPolicyError("queryPerSeconds parameter not found in configmap " + configMap.Name)
+	}
+
+	qpsConverted, err := strconv.ParseFloat(qpsRead, 32)
+
+	if err != nil {
+		return nil, NewTuningPolicyError(fmt.Sprintf("failed to convert queryPerSeconds parameter to float; " + err.Error()))
```

Comment:
Not sure there is reason to use `fmt.Sprintf` for a single string w/o parameters. Please fix to one of:
```suggestion
		return nil, NewTuningPolicyError("failed to convert queryPerSeconds parameter to float; " + err.Error())
```

Or (preferred):
```suggestion
		return nil, NewTuningPolicyError(fmt.Sprintf("failed to convert queryPerSeconds parameter to float; %s", err.Error()))
```

Reply:
I have used the second option. Check if it is fine now.